internal/pkg/cli/storage_init_test.go

```diff
@@ -632,10 +664,18 @@ func TestStorageInitOpts_Ask(t *testing.T) {
 			} else {
 				require.Nil(t, err)
 			}
+			if tc.wantedVars != nil {
+				tc.wantedVars.prompt = opts.prompt
+				require.Equal(t, *tc.wantedVars, opts.initStorageVars)
+			}
 		})
 	}
 }
 
+func TestStorageInitOpts_AskOutputs(t *testing.T) {
+
+}
```

Comment:
why do we have this empty test func?

Reply:
Sorry, fixed. Added while I was looking for the best way to validate this and forgot to remove.